File: sandbox/php/src/TeamDeleteExample.php
<?php

namespace Dropbox\SignSandbox;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use Dropbox;

$config = Dropbox\Sign\Configuration::getDefaultConfiguration();
$config->setUsername("YOUR_API_KEY");
// $config->setAccessToken("YOUR_ACCESS_TOKEN");

try {
    (new Dropbox\Sign\Api\TeamApi(config: $config))->teamDelete();
} catch (Dropbox\Sign\ApiException $e) {
    echo "Exception when calling TeamApi#teamDelete: {$e->getMessage()}";
}

File: examples/TeamDeleteExample.py
import json
from datetime import date, datetime
from pprint import pprint

from dropbox_sign import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    username="YOUR_API_KEY",
    # access_token="YOUR_ACCESS_TOKEN",
)

with ApiClient(configuration) as api_client:
    try:
        api.TeamApi(api_client).team_delete()
    except ApiException as e:
        print("Exception when calling TeamApi#team_delete: %s\n" % e)
